hack/lib/util/golang.sh

```diff
@@ -21,3 +20,17 @@ function os::golang::verify_go_version() {
 	fi
 }
 readonly -f os::golang::verify_go_version
+
+# os::golang::verify_glide_version ensures the glide is at the required level
+function os::golang::verify_glide_version() {
+	os::util::ensure::system_binary_exists 'glide'
+
+	local glide_version
+	glide_version=($(glide --version))
+	if ! echo "${glide_version[2]#v}" | awk -F. -v min=$OS_GLIDE_MINOR_VERSION '{ exit $2 < min }'; then
```

Comment:
Seems to do what you want, even on Mac, but it's ugly. And let's hope there is never a Glide 1.0.

Reply:
I know, but that's the best I could come up with :/ I hope that by then dep will be mature enough ;)